test/unit/requestMatchingTest.js

```diff
@@ -212,7 +213,7 @@ describe('Request Matching middleware', () => {
     let addedChannelNames = []
 
     afterEach(() => ChannelModel.deleteMany({name: { $in: addedChannelNames }}))
-
+/*
```

Comment:
The integration tests won't work anymore because content matching isn't being performed in this method call (matchRequest) anymore. There are unit tests specifically for content matching that are still in place.